Resources/Locale/en-US/weapons/melee/melee.ftl

```diff
@@ -1 +1,3 @@
 melee-inject-failed-hardsuit = Your {$weapon} cannot inject through hardsuits!
+melee-telebaton-examined-extended = A compact and elegant tool for subjugating your staff when they feel like not taking orders. Activate it in your hand to retract.
+melee-telebaton-examined-retracted = A compact and elegant tool for subjugating your staff when they feel like not taking orders. Activate it in your hand to extend.
```

Comment:
These should just be set up like this ( the normal desc is already in the prototype)

```suggestion
melee-telebaton-examined-extended = Activate it in your hand to retract.

melee-telebaton-examined-retracted = Activate it in your hand to extend.
```